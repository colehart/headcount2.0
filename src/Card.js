import React from 'react'

const Card = () => {
  return(
    <article className="Card">
    </article>
  )
}

export default Card