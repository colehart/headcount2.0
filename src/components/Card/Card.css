/*CARD COMPONENT RELATED CSS GOES HERE*/
.Card {
  height: 300px;
  width: 200px;
  margin: 5px 0;
  padding: 0.5rem;
  display: grid;
  grid-template-columns: 1fr;
  grid-template-rows: 4rem 1fr;
  justify-items: center;
  color: #006989;
  letter-spacing: 0.5px;
  border-radius: 5%;
  background: rgb(255, 255, 255);
  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
}

h2 {
  text-align: center;
  grid-row: 1 / 2;
  font-family: "Roboto Condensed", sans-serif;
  font-weight: 700;
  font-size: 20px;
  margin: 0;
  padding-top: 1.25rem;
}

ul {
  grid-row: 2 / 3;
  list-style-type: none;
  font-family: "Roboto", sans-serif;
  font-weight: 400;
  text-size: 1.5rem;
  padding-left: 0;
  align-self: center;
}

.low {
  color: #EF713B;
}