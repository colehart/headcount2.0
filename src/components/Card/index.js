import React from 'react'
import './Card.css'

const Card = ({location, data}) => {
  return(
    <article className="Card">
      <h2>{location}</h2>
      <ul>
        { Object.keys(data).map((year, index) => {
          return (<li key={index}>
            {year}: {data[year]}
          </li>)
          })
        }
      </ul>
    </article>
  )
}

export default Card