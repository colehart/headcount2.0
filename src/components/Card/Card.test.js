import React from 'react'
import { shallow } from 'enzyme'
import Card from './index'

describe('Card', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Card />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})