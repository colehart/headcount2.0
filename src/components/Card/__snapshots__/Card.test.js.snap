// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match the snapshot 1`] = `
<article
  className="Card"
>
  <h2>
    COLORADO
  </h2>
  <ul>
    <li
      key="0"
    >
      2018
      : 
      1
    </li>
  </ul>
</article>
`;
